src/main/scala/zio/prelude/CommutativeBoth.scala

```diff
@@ -49,14 +55,18 @@ object CommutativeBoth extends LawfulF.Invariant[CommutativeBothDeriveEqualInvar
   implicit def ChunkCommutativeBoth: CommutativeBoth[Chunk] =
     new CommutativeBoth[Chunk] {
       def both[A, B](fa: => Chunk[A], fb: => Chunk[B]): Chunk[(A, B)] = fa zip fb
+
+      def bothPar[A, B](fa: => Chunk[A], fb: => Chunk[B]): Chunk[(A, B)] = fa zip fb
```

Comment:
In cases like this, I'm thinking whether it wouldn't be for the best if we reused `both` method to emphasize that the implementation is literally the same, like
```scala
       def bothPar[A, B](fa: => Chunk[A], fb: => Chunk[B]): Chunk[(A, B)] = both(fa, fb) 
```